aeron-driver/src/main/java/io/aeron/driver/DriverConductor.java

```diff
@@ -1764,15 +1819,66 @@ private NetworkPublication newNetworkPublication(
         }
     }
 
+
+    private static String enrichNetworkPublicationUri(ChannelUri channelUri, int streamId, int sessionId, int initialTermId, SendChannelEndpoint channelEndpoint, PublicationParams params)
```

Comment:
You could create a base enrich method that sets common parameters such as `mtu`, `term-length`, `initial-term-id` (etc.) and will return a `ChannelUriStringBuilder` so that it can be further enhanced.

Reply:
Done